integrations-and-supported-tools/kedro/scripts/kedro_neptune_advanced/src/requirements.txt

```diff
@@ -1,14 +1,14 @@
-black==21.5b1
-flake8>=3.7.9, <4.0
-ipython~=7.0
-isort~=5.0
-jupyter~=1.0
-jupyter_client>=5.1, <7.0
-jupyterlab~=3.0
-kedro[pandas.CSVDataSet]==0.17.4
-kedro-telemetry~=0.1.0
-nbstripout~=0.4
-pytest-cov~=2.5
-pytest-mock>=1.7.1, <2.0
-pytest~=6.2
-wheel>=0.35, <0.37
+black
+flake8
+ipython
+isort
+jupyter
+jupyter_client
+jupyterlab
+kedro[pandas.CSVDataSet]
+kedro-telemetry
```

Comment:
For kedro, it's dangerous to unpin it they have huge changes rather quickly.

Reply:
will update the kedro project to be in sync with kedro version in an upcoming PR. Pinning it will lead us to showcase outdated versions which might have reached EOL

Reply:
Great!